src/internet_identity/internet_identity.did

```diff
@@ -244,8 +255,9 @@ service : (opt InternetIdentityInit) -> {
     remove : (UserNumber, DeviceKey) -> ();
     // Returns all devices of the user (authentication and recovery) but no information about device registrations.
     // Note: Clears out the 'alias' fields on the devices. Use 'get_anchor_info' to obtain the full information.
-    // Note: Will be changed in the future to be more consistent with get_anchor_info.
+    // Deprecated: Use 'get_device_credentials' instead.
     lookup : (UserNumber) -> (vec DeviceData) query;
+    get_device_credentials : (UserNumber) -> (GetDeviceCredentialsResponse) query;
```

Comment:
I would suggest naming this `get_credentials` (or if really necessary `get_anchor_credentials`), I started looking at this assuming it would somehow only list the credentials for a device

Reply:
Makes sense. I have renamed it `get_anchor_credentials`.